lib/c-api/tests/wasm.h

```diff
@@ -0,0 +1,49 @@
+// This header file is used only for test purposes! It is used by unit
+// test inside the `src/` directory for the moment.
+
+#ifndef TEST_WASM
+#define TEST_WASM
+
+#include "../wasm.h"
+#include "../wasmer_wasm.h"
+#include <stdio.h>
+#include <string.h>
+
+wasm_engine_t *wasm_engine_new() {
+  wasm_config_t *config = wasm_config_new();
+
+  char *wasmer_test_compiler = getenv("TEST");
```

Comment:
Can we rename this variable as `WASMER_C_API_TEST_CONFIGURATION`, or something like this; because just `TEST` is meaningless :-). Thanks!

Reply:
I agree, also `TEST` has a high risk of collision, having `WASMER_` in front reduces the chances of random looking breakage in case that env var already exists

Reply:
`WASMER_C_API_TEST_CONFIGURATION` is like insanely large. I usually was testing by hand and typing that much was a pain. But I can prefix it with `WASMER_CAPI_CONFIG`

Reply:
Fixed